tests/unit/test_wheel_builder.py

```diff
@@ -52,6 +52,7 @@ def __init__(
         self.use_pep517 = use_pep517
         self._supports_pyproject_editable = supports_pyproject_editable
 
+    @property
     def supports_pyproject_editable(self) -> bool:
         return self._supports_pyproject_editable
```

Comment:
We can simply rename the attribute instead. Or I think the entire class can be converted to dataclass!